lexicon/providers/rcodezero.py

```diff
@@ -0,0 +1,196 @@
+"""Module provider for RcodeZero"""
+from __future__ import absolute_import
+import json
+import logging
+
+import requests
+from lexicon.providers.base import Provider as BaseProvider
+
+
+LOGGER = logging.getLogger(__name__)
+
+NAMESERVER_DOMAINS = ['rcode0.net']
+
+
+def provider_parser(subparser):
+    """Return the parser for this provider"""
+    subparser.add_argument(
+        "--auth-token", help="specify token for authentication")
+
+
+class Provider(BaseProvider):
+    """Provider class for Cloudflare"""
+    def __init__(self, config):
+        super(Provider, self).__init__(config)
+        self.domain_id = None
+        self._zone_data = None
+        self.api_endpoint = 'https://my.rcodezero.at/api/v1'
+
+    def _authenticate(self):
+        if self._zone_data is None:
+            self._zone_data = self._get('/zones/' +self.domain)
+
+        self.domain_id = self.domain
+
+    # Create record. If record already exists with the same content, do nothing'
```

Comment:
Technically, the API is always called, even if the record already exists, but will have no effect on the existing record set. Could you update this inline comment accordingly?

Reply:
fixed in 45e67a5